.github/workflows/helm-ci.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
       - name: Setup chart testing
         uses: helm/chart-testing-action@v2.0.1
       - name: Run chart-testing (lint)
-        run: ct lint
+        run: ct lint --chart-dirs helm --all
```

Comment:
Shouldn't be `helm/steward`?

Reply:
No, it needs only list of dirs containing helm charts, not the helm charts.